packages/nitro-protocol/package.json

```diff
@@ -35,7 +35,7 @@
     "build:ci": "yarn build",
     "test:ci": "yarn test",
     "test:typescript": "npx jest --testPathIgnorePatterns='test/contracts' --reporters='default'",
-    "test": "yarn prettier:check && yarn clearContracts && npx ganache-then-jest --runInBand",
+    "test": "yarn lint:check && yarn clearContracts && npx ganache-then-jest --runInBand",
```

Comment:
Can we decouple linting from testing? They are separate concerns.

Reply:
Going to do this in a separate PR